lib/pure/json.nim

```diff
@@ -949,10 +949,50 @@ proc pretty*(node: JsonNode, indent = 2): string =
   result = ""
   toPretty(result, node, indent)
 
+proc toUgly*(result: var string, node: JsonNode) =
+  ## Converts `node` to its JSON Representation, without
+  ## regard for human readability.
```

Comment:
Mention in a comment here that this function has been written to make `$` faster.